app/payment/util/refund_form_mail.py

```diff
@@ -0,0 +1,198 @@
+import base64
+import functools
+import io
+import operator
+import tempfile
+from datetime import datetime
+
+from django.core.mail import EmailMultiAlternatives
+
+# Handle HEIC photoes
+import pyheif
+from fpdf import FPDF
+# Handle PDF files
+from pdf2image import convert_from_path
+from PIL import Image
+
+
+class UnsupportedFileException(Exception):
+    pass
+
+
+field_title_map = {
+    "date": "Dato:",
+    "committee": "Gruppe/utvalg:",

```

Comment:
Does this mean that just groups kan ask for refund

Reply:
yes